apps/web/src/pages/templates/components/VariantItemCard.tsx

```diff
@@ -0,0 +1,32 @@
+import { StepTypeEnum } from '@novu/shared';
+import { useNavigate, useParams } from 'react-router-dom';
+
+import { stepIcon } from '../constants';
+import { useBasePath } from '../hooks/useBasePath';
+import { useStepSubtitle } from '../hooks/useStepSubtitle';
+import { WorkflowNode } from '../workflow/workflow/node-types/WorkflowNode';
+
+export const VariantItemCard = ({ variant }) => {
+  const { channel, stepUuid = '' } = useParams<{
+    channel: StepTypeEnum;
+    stepUuid: string;
+  }>();
+  const subtitle = useStepSubtitle(variant, channel);
```

Comment:
reuse the same logic that builds the subtitle for the step in the variant list item card